.gitignore

```diff
@@ -1,12 +1,11 @@
 .idea
 *.iml
-dist/
```

Comment:
Shouldn't `dist` still be ignored?  We're adding a lot of compiled files in this PR.